src/main/java/sirius/biz/jobs/batch/file/FileImportJob.java

```diff
@@ -183,10 +182,15 @@ private static Boolean hasNoArchiveFile(Map<String, String> params) {
      * Helps to create a custom file parameter with an appropriate file extension filter.
      *
      * @param acceptedFileExtensions the file extensions to accept
+     * @param description            the description to use in the parameter
      * @return the parameter used to select the import file
      */
-    public static Parameter<VirtualFile> createFileParameter(@Nullable List<String> acceptedFileExtensions) {
+    public static Parameter<VirtualFile> createFileParameter(@Nullable List<String> acceptedFileExtensions,
+                                                             String description) {
```

Comment:
Maybe switch the nullable annotation here? Or completely remove it? My expectation is that there is no need for collection parameters to be null at API level. Internal robustness remains of course :-)

Reply:
I opt for the first. When collecting, makes definitely no sense. But the creation method is also used in job instances, where its goal is just to obtain the parameter's value, and in there we need no description.